get_genotype_vcf.py

```diff
@@ -162,7 +162,7 @@ def add_remove_chr_and_index_job(vcf_path):
     )
     bcftools_job.command(
         f"""
-        bcftools annotate --rename-chrs chr_update.txt {vcf_input} | \\
+        bcftools annotate --rename-chrs chr_update.txt --set-id +'%CHROM\_%POS\_%REF\_%FIRST_ALT' {vcf_input} | \\
```

Comment:
Just flagging that the documentation says that the `+` at the start of the `--set-id` command then only missing IDs will be set. So if not all IDs are missing then there will be different formatting for the ID field in the vcf file

Reply:
good flag! I think the way Hail exports to vcf it would only add an ID if there were [`rsids`](https://www.pharmgkb.org/page/glossary#:~:text=rsID%20number,to%20a%20specific%20genetic%20variant.) in the VDS / MT object, which we specifically are avoiding doing, because they can be ambiguous compared to just specifying the chromosome, position and reference / alternative allele. So it will always be missing for us, or at least for the cohorts included in this project.